countries_info/src/rest_countries_client.py

```diff
@@ -24,6 +24,7 @@ def fetch_country_info(self, country_name: str) -> CountryInfoSchema:
                 capital=country_info_json.get("capital", []),
                 languages=list(country_info_json.get("languages", {}).values()),
                 population=country_info_json.get("population", 0),
+                biggest_city=country_info_json.get("biggest_city", ""),
```

Comment:
```suggestion
                biggest_city=country_info_json.get("biggest_city"),
```

Consider using `None` instead of any empty string